datarequest.py

```diff
@@ -573,24 +573,31 @@ def api_datarequest_datamanager_review_get(ctx, request_id):
 
 @api.make()
 def api_datarequest_is_owner(ctx, request_id, user_name):
-    result = datarequest_is_owner(ctx, request_id, user_name)
+    """Check if the invoking user is also the owner of a given data request
 
-    if not (result['status'] == 0):
-        return api.Error("error", "Something went wrong in determining datarequest ownership")
+       :param request_id: Unique identifier of the data request
+       :type request_id: str
+       :param user_name: Username of the user whose ownership is checked
+       :type user_name: str
 
-    return result['owner']
+       :raises Exception: It was not possible to unambiguously determine the owner of the data request (either 0 or > 1 results for the data request)
+       :return: `True` if ``user_name`` matches that of the owner of the data request with id ``request_id``, `False` otherwise
+       :rtype: bool
+    """
+    return datarequest_is_owner(ctx, request_id, user_name)
```

Comment:
Can you catch the Exception here and return some extra information with api.Error. For example: https://github.com/UtrechtUniversity/irods-ruleset-uu/blob/f6c7a6de6f5aa4c22b6a4b6e1a5a89916d57e1bb/datarequest.py#L199